package.json

```diff
@@ -41,14 +41,11 @@
   },
   "pnpm": {
     "patchedDependencies": {
-      "@docusaurus/theme-search-algolia@2.3.1": "patches/@docusaurus__theme-search-algolia@2.3.1.patch"
+      "@docusaurus/theme-search-algolia@3.0.0-alpha.0": "patches/@docusaurus__theme-search-algolia@2.3.1.patch"
```

Comment:
this patch seemed to apply cleanly, and still is having the intended effect in local testing, so that's a relief 😅